src/components/TeamGeneratorPage/TeamGenerator.js

```diff
@@ -11,10 +12,17 @@ const TeamGenerator = () => {
 	};
 
 	function importPlayerList(importPlayers) {
-		const set = new Set([...playerList, ...importPlayers]);
-		setPlayerList([...set]);
+		const playersSet = new Set([...playerList, ...importPlayers]);
+		const players = [...playersSet];
+		setStoredPlayersList(players);
+		setPlayerList(players);
 	}
 
+	useState(() => {
```

Comment:
What is that useState???

Reply:
fixed in https://github.com/AriGute/TeamsGenerator/pull/3